src/series/groupedSeries.js

```diff
@@ -37,55 +37,69 @@ export default function() {
 
             g.select('g')
                 .datum(values)
-                .each(function(series, index) {
+                .each(function(_series, index) {
                     var container = d3.select(this);
 
                     // create a composite scale that applies the required offset
                     var compositeScale = function(_x) {
                         return xScale(_x) + offsetScale(index);
                     };
-                    bar.xScale(compositeScale);
+                    series.xScale(compositeScale);
 
                     // adapt the decorate function to give each series the correct index
-                    bar.decorate(function(s, d) {
+                    series.decorate(function(s, d) {
                         decorate(s, d, index);
                     });
 
-                    container.call(bar);
+                    container.call(series);
                 });
         });
     };
 
-    groupedBar.groupWidth = function(_x) {
+    groupedSeries.groupWidth = function(_x) {
         if (!arguments.length) {
             return barWidth;
         }
         barWidth = d3.functor(_x);
-        return groupedBar;
+        return groupedSeries;
     };
-    groupedBar.decorate = function(_x) {
+    groupedSeries.decorate = function(_x) {
         if (!arguments.length) {
             return decorate;
         }
         decorate = _x;
-        return groupedBar;
+        return groupedSeries;
     };
-    groupedBar.xScale = function(_x) {
+    groupedSeries.xScale = function(_x) {
         if (!arguments.length) {
             return xScale;
         }
         xScale = _x;
-        return groupedBar;
+        return groupedSeries;
     };
-    groupedBar.values = function(_x) {
+    groupedSeries.values = function(_x) {
         if (!arguments.length) {
             return values;
         }
         values = _x;
-        return groupedBar;
+        return groupedSeries;
+    };
+    groupedSeries.value = function(_x) {
+        if (!arguments.length) {
+            return value;
+        }
+        value = _x;
+        return groupedSeries;
+    };
+    groupedSeries.offsetScale = function(_x) {
+        if (!arguments.length) {
+            return offsetScale;
+        }
+        offsetScale = _x;
+        return groupedSeries;
     };
 
-    d3.rebind(groupedBar, bar, 'yValue', 'xValue', 'yScale');
+    d3.rebind(groupedSeries, series, 'yScale');
```

Comment:
How about using `rebindAll` that way it will dynamically rebind the series properties. Not sure if this might cause name collisions? Definitely worth considering.